src/main/java/net/datafaker/providers/base/BaseFaker.java

```diff
@@ -298,6 +300,16 @@ public void addUrl(Locale locale, URL url) {
         fakeValuesService().addUrl(locale, url);
     }
 
+    public static <T> T populate(Class<T> clazz) {
+        var fakeFactory = new FakeResolver<>(clazz);
+        return fakeFactory.get(null);
```

Comment:
Not sure that `get` is a right name here.... Probably something like `generate` would be better since it really generates